plugins/ingest-attachment/build.gradle

```diff
@@ -23,7 +23,7 @@ esplugin {
 }
 
 versions << [
-  'tika': '1.17',
+  'tika': '1.18',
   'pdfbox': '2.0.8',
```

Comment:
I think we need to check if these have been bumped in `tika` itself as we don't pull in transitive dependencies automatically. If `tika` bumped its dependency versions here, we should too. The same goes for `bouncycastle` and `poi`.

Reply:
Jackson is at 2.9.5, but it seems like we're at 2.8.10?  Is there anything I can/should do about this?

For the others --

* Updated pdfbox 2.08 -> 2.09
* Bouncycastle is already at 1.55 and the compile dependency is 1.54 (I assume higher is okay?)
* poi and mime4j are already the same
* org.tukaani:xz 1.6 -> 1.8
*commons-io-commons-io 2.5 -> 2.6
* org.apache.commons:commons-compress 1.14 -> 1.16.1

I only checked against the versions in the gradle file specifically for ingest-attachment.  Are there any other places I need to check?

Reply:
The Jackson one is tricky since we inherit the dependency from core and we are version locked there. I think that means we need to leave that one as-is for now. The rest look good to be but I think we should take Bouncy Castle to 1.54 since the Tika [POM does not say 1.54+](https://github.com/apache/tika/blob/1.18/tika-parsers/pom.xml#L52). I would rather play it safe.